controller.js

```diff
@@ -0,0 +1,50 @@
+var calc = new Calculator()
+window.onload = document
+  .getElementById( 'calculator-output' )
+  .innerHTML = calc.getDisplay()
+
+function clickHandler(event) {
+  var displayScreen = document.getElementById( 'calculator-output' )
+  var keystroke = event.target.dataset.value
+  console.log( 'keystroke', keystroke )
+  var displayLength = calc.getDisplayLength()
+
+  sendInputToCalculator( keystroke )
+  checkForAllClear( keystroke, displayScreen )
+  adjustDisplayFont( displayLength, displayScreen )
+  checkForOverFlowingDisplay( displayLength, displayScreen )
```

Comment:
Just to clarify for my understanding, it looks like the three functions after `sendInputToCalculator` are there explicitly to update the View.  Is that correct?